app/controllers/audits_controller.rb

```diff
@@ -40,6 +40,7 @@ def finalize
     ActiveRecord::Base.transaction do
       @audit.storage_location.increase_inventory increasing_adjustment
       @audit.storage_location.decrease_inventory decreasing_adjustment
+      AuditEvent.publish(@audit)
```

Comment:
I'm worried that the `@audit.split_differences` routine does weird things, should this event record be above the split_difference, or even higher up than that near the `@audit.save`?

Reply:
It's in a transaction, so it doesn't really matter. Either all of the records will be created or none of them will.